### DISCUSSION TITLE
Mend Renovate dashboard just says "Failed" with no log

### DISCUSSION THREAD

**Discussion by Jamesking56** [renovatebot/renovate#30151] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Mend Renovate hosted app on github.com

### Please tell us more about your question or problem

https://developer.mend.io/github/PMP-Developments/pmprp-townhall

I've just onboarded this repo and it now just states "Failed" with no log to tell me why or whats failed.

**Comment by ysoldak** [comment]
Same here. We have a repository with many "npm" and "github-actions" dependencies.
Runs take lot of time and all fail eventually. No log can be seen though, not for failed not for running jobs.

One of the latest failed jobs id: b4a31e4e-ee1e-4285-a2ec-121c93afa41d

**Reply by nabeelsaabna** [reply]
The run exceeded the maximum RAM
```
FATAL ERROR: Reached heap limit Allocation failed - JavaScript heap out of memory
```

**Reply by nabeelsaabna** [reply]
the last passing jobs:
```
2024-06-17 12:14:29.971210 +00:00
2024-06-17 11:56:01.733018 +00:00
2024-06-17 11:35:00.099427 +00:00
```

then starting from `2024-06-17 14:20:59.600678 +00:00` all jobs are failing for the same repository

**Reply by rarkins** [reply]
@nabeelsaabna does that coincide with any CLI version upgrade?

**Reply by nabeelsaabna** [reply]
Actually we did update from `37.393.0` to `37.410.1` on `17 Jun 2024 16:34:03`  

Note: there is a 3h time difference between my github time and database time, so it's indeed matching

**Reply by rarkins** [reply]
Any general increase in memory usage observable?

**Reply by nabeelsaabna** [reply]
I can schedule a new job for this repo with renovate `37.393.0` to confirm 
is that ok ?

**Reply by stasostrovskyi** [reply]
Yes, please! (I am on the same team as @ysoldak)

**Reply by ysoldak** [reply]
I'm back from vacay and can confirm the jobs still failing. Latest failed job: 0190fd13-8422-7be6-84be-cc75fe789356

**Reply by ysoldak** [reply]
@nabeelsaabna @rarkins let me know if we shall move the discussion somewhere else, a separate ticket, perhaps?

**Reply by HonkingGoose** [reply]
@ysoldak If you still have problems, please open a new _Request help_ discussion. Please copy/paste the relevant information into the new Discussion too.

**Reply by ysoldak** [reply]
Thank you, @HonkingGoose 
Link to the separate help discussion: https://github.com/renovatebot/renovate/discussions/30499

**Comment by nabeelsaabna** [comment, marked as answer]
@Jamesking56 it seems to be a temporary issue related to github.com availability, I see the many successful job runs including the last one

**Reply by Jamesking56** [reply]
Yep working now cheers
